Description: uncomment "TasksMax" in systemd service file (since our version of systemd supports it)
Author: Tianon Gravi <tianon@debian.org>
Origin: https://github.com/docker/docker/pull/21491#issuecomment-201431374

diff --git a/contrib/init/systemd/docker.service b/contrib/init/systemd/docker.service
index c10655d..d14b320 100644
--- a/contrib/init/systemd/docker.service
+++ b/contrib/init/systemd/docker.service
@@ -19,7 +19,7 @@ LimitNPROC=infinity
 LimitCORE=infinity
 # Uncomment TasksMax if your systemd version supports it.
 # Only systemd 226 and above support this version.
-#TasksMax=infinity
+TasksMax=infinity
 TimeoutStartSec=0
 # set delegate yes so that systemd does not reset the cgroups of docker containers
 Delegate=yes
